js/src/Ice/TraceUtil.js

```diff
@@ -58,90 +58,30 @@ function printReply(s, inputStream) {
     const requestId = inputStream.readInt();
     s.push(`\nrequest id = ${requestId}`);
 
+    // This creates a new enumerator (e.g. "18": 18) if one doesn't exist yet.
     const replyStatus = ReplyStatus.valueOf(inputStream.readByte());
-    s.push(`\nreply status = ${replyStatus.value} `);
+    s.push(`\nreply status = ${replyStatus.name}`);
```

Comment:
We provide toString that returns name

```suggestion
    s.push(`\nreply status = ${replyStatus}`);
```

Reply:
Fixed